core/src/test/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/function/arrayfunction/ArrayStarExpressionTest.kt

```diff
@@ -0,0 +1,128 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.function.arrayfunction
+
+import ch.ergon.dope.DopeParameters
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.DopeQueryManager
+import ch.ergon.dope.helper.ManagerDependentTest
+import ch.ergon.dope.helper.someObjectArrayField
+import ch.ergon.dope.helper.someObjectField
+import ch.ergon.dope.helper.someObjectSelectRawClause
+import ch.ergon.dope.resolvable.expression.unaliased.type.asParameter
+import ch.ergon.dope.resolvable.expression.unaliased.type.toDopeType
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class ArrayStarExpressionTest : ManagerDependentTest {
+    override lateinit var manager: DopeQueryManager
+
+    @Test
+    fun `should support ARRAY_STAR`() {
+        val expected = DopeQuery(
+            queryString = "ARRAY_STAR(`objectArrayField`)",
+        )
+        val underTest = ArrayStarExpression(someObjectArrayField())
+
+        val actual = underTest.toDopeQuery(manager)
+
+        assertEquals(expected, actual)
+    }
+
+    @Test
+    fun `should support ARRAY_STAR with named parameter`() {
+        val parameterName = "objectArray"
+        val parameterValue = listOf(
```

Comment:
always use the plural of the variable name if the type is a list (parameterValue**s**)

Reply:
it is singular because it represents the value of one single Parameter